habitat_baselines/config/instance_imagenav/ddppo_instance_imagenav.yaml

```diff
@@ -0,0 +1,75 @@
+BASE_TASK_CONFIG_PATH: configs/tasks/instance_imagenav_hm3d.yaml
+CMD_TRAILING_OPTS: ["TASK_CONFIG.ENVIRONMENT.ITERATOR_OPTIONS.MAX_SCENE_REPEAT_STEPS", "50000"]
+SIMULATOR_GPU_ID: 0
+TORCH_GPU_ID: 0
+VIDEO_OPTION: []
+TENSORBOARD_DIR: tb
+VIDEO_DIR: video_dir
+TEST_EPISODE_COUNT: -1
+EVAL_CKPT_PATH_DIR: data/new_checkpoints
+NUM_ENVIRONMENTS: 4
+CHECKPOINT_FOLDER: data/new_checkpoints
+TRAINER_NAME: ddppo
+SENSORS: [DEPTH_SENSOR, RGB_SENSOR]
+NUM_UPDATES: 270000
```

Comment:
This number feels a bit ... exact ? Can you set it to 200,000 or 500,000 if possible?